src/core/mac/mac_frame.hpp

```diff
@@ -34,6 +34,8 @@
 #ifndef MAC_FRAME_HPP_
 #define MAC_FRAME_HPP_
 
+#include <openthread-core-config.h>
```

Comment:
Change this to:

```
#ifdef OPENTHREAD_CONFIG_FILE
#include OPENTHREAD_CONFIG_FILE
#else
#include <openthread-config.h>
#endif
```

Example: [src/core/openthread.cpp](https://github.com/openthread/openthread/blob/master/src/core/openthread.cpp)